### ISSUE TITLE
ตัวอย่าง

### ISSUE THREAD

**Issue by Tuaa** [Tuaa/Hackathon-Mobile-ML#1]
``` python
import tensorflow as tf
import numpy as np
from tensorflow import keras

from tensorflow.python.platform import tf_logging as logging
logging.set_verbosity(logging.ERROR)
print(tf.__version__)
      
model = tf.keras.Sequential([keras.layers.Dense(units=1, input_shape=[1])])
      
model.compile(loss='mean_squared_error',
              optimizer=tf.keras.optimizers.Adam(0.1))

# อนุกรม
xs = np.array([-40, -10,  0,  8, 15, 22,  38],  dtype=float)
ys = np.array([-40,  14, 32, 46, 59, 72, 100],  dtype=float)

model.fit(xs, ys, epochs=500) # รอบ
